en/docs/develop/understand-ide/editors/flow-diagram-editor/ai.md

```
@@ -86,7 +90,11 @@ For the ingestion flow, supported formats, and how to point the loader at a dire
 
 ![Augment Query form with Context, Query, Result, and Result Type fields](/img/develop/flow-design-elements/augment-query-form.png)
 
+<<<<<<< fix-broken-links
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)